include/triton-shared/Conversion/TritonArithToLinalg/ConversionPatterns.hpp

```diff
@@ -2255,4 +2257,4 @@ static void populateExternElementwiseOpToMLIROps(RewritePatternSet &patterns) {
 
 } // namespace
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
This have nothing to do with our requirement, I think revert it to keep the change as small as possible is better.